### ISSUE TITLE
Publish app to Google Playstore, F-Droid and the website

### ISSUE THREAD

**Issue by SailReal** [SailReal/nina-foss#11]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/SailReal/nina-foss/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/SailReal/nina-foss/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

Publish app to Google Playstore, F-Droid and the website

### Motivation

As a user, I want to be able to download this app using Google Playstore, F-Droid and the website https://nina-foss.de/

### Anything else?

- Create app description and screenshots
- Adjust Fastlane script to this project
- Create request for packaging on https://gitlab.com/fdroid/rfp/-/issues/

**Comment by Poussinou** [comment]
I opened an Issue on F-Droid side [here](https://gitlab.com/fdroid/rfp/-/issues/2038). We are waiting for Fastlane (and eventually a FUNDING.yml if you want one, F-Droid will take the donate information directly from this file). The use of [Releases](https://github.com/SailReal/nina-foss/releases) would also probably be a good idea ;)

Also, our bot will give us some information later, don't hesitate to check the Gitlab Issue later.

**Comment by SailReal** [comment]
@Poussinou thank you very much for your work. I must admit that this app is in an very early alpha state, I'm using it myself since months but I don't think that anyone else would be satisfied with it, especially because https://github.com/SailReal/nina-foss/issues/5 isn't finished. The reason I paused spending time with it is that as I started, no other app was available but later https://github.com/nucleus-ffm/foss_warn appeared and in my opinion, one FOSS version of NINA is enough so I paused the development and put my time into other projects.

TLDR; I'm not quite sure yet if and how to proceed with the project.
